src/lib/sw_engine/tvgSwRle.cpp

```diff
@@ -923,8 +923,9 @@ void updateRleSpans(SwRleData *rle, const SwSpan* curSpans, uint32_t size)
     }
 
     if (!rle->spans || !curSpans) return;
+    rle->alloc = size * 2;
```

Comment:
why 2? it could increase unnecessary memory for shape drawings,  we need to examine the exact reason